### PR TITLE
build: simplify release process

### PR DESCRIPTION
**Pull request by bluestreak01**
On a branch (critical)

1. tag the release & upload to maven central
```
mvn -B release:prepare release:perform
```

2. Release binary from Git tag (lets assume tag is `7.0.4`)
```
git fetch
git checkout tags/7.0.4
mvn clean deploy -DskipTests -P build-web-console,maven-central-release
```

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -22,14 +22,14 @@
...
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <name>QuestDB core</name>
     <description>QuestDB a is high performance SQL time series database</description>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.javadoc.skip>false</maven.javadoc.skip>
```
Comment (jerrinot) [inline core/pom.xml:32]: why is this needed? isnt this default? the `maven-central-release` profile add the JavaDoc plugin.

**Review by jerrinot — APPROVED** [review]
looks good!

File: core/pom.xml (hunk 1 above)
Comment (bluestreak01) [reply core/pom.xml:32]: this has to be retested with release, i remember a failure to release where javadoc was not generated.
